src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -248,11 +248,23 @@ function qn_svdTTN(
   root_vertex::VT;
   kwargs...,
 )::TTN where {C,VT}
+  
   mindim::Int = get(kwargs, :mindim, 1)
   maxdim::Int = get(kwargs, :maxdim, 10000)
   cutoff::Float64 = get(kwargs, :cutoff, 1e-15)
-
   #ValType = ITensors.determineValType(ITensors.terms(os))  #now included as argument in function signature
+  
+  # check for qns on the site indices
+  #FIXME: this check for whether or not any of the siteindices has QNs is somewhat ugly
+  #FIXME: how are sites handled where some sites have QNs and some don't?
+  
+  thishasqns=false
+  for site in vertices(sites)
+    if hasqns(sites[site])
+      thishasqns=true
+      break
+    end
+  end
```

Comment:
You should be able to do something like `any(v -> hasqns(sites[v]), vertices(sites))`.

Reply:
but generally the logic is that if a single site-index has QNs, the link indices are also supposed to have QNs everywhere, right?

Reply:
Well, basically right now it doesn't work to have mixed QNs or not QNs, so implicitly we are assuming it is all or nothing.

Reply:
Probably we could do something smarter where we add trivial QNs to indices without QNs, if any have QNs, but that's outside the scope of this PR.